commit a2a5a86ac0500d5e9f463377d1b9220f244d6624
Author: Xavier Bachelot <xavier@bachelot.org>
Date:   Fri Sep 29 18:07:33 2017 +0200

    Allow multiple attrs in include_ldap_2level_query

diff --git a/src/lib/Sympa/List.pm b/src/lib/Sympa/List.pm
index 7cdc5f825..6459add0b 100644
--- a/src/lib/Sympa/List.pm
+++ b/src/lib/Sympa/List.pm
@@ -5568,7 +5568,7 @@ sub _include_users_ldap_2level {
             'search',
             base   => "$suffix2",
             filter => "$filter2",
-            attrs  => ["$ldap_attrs2"],    # FIXME: multiple attrs?
+            attrs  => [@ldap_attrs2],
             scope  => "$ldap_scope2"
         );
         unless ($mesg) {
