commit 02e4973a9fd6b2197a26730161d2ca077ae43804
Author: IKEDA Soji <ikeda@conversion.co.jp>
Date:   Tue Sep 5 13:06:31 2017 +0900

    Issue #50: smtpc build warnings due to malicious typo.

diff --git a/src/smtpc/smtpc.c b/src/smtpc/smtpc.c
index 1ffffc210..91aa7be5a 100644
--- a/src/smtpc/smtpc.c
+++ b/src/smtpc/smtpc.c
@@ -226,7 +226,7 @@ static int parse_options(int *argcptr, char ***argvptr)
 			return -1;
 		    }
 
-		    if (*p == ':' && ++p != '\0')
+		    if (*p == ':' && *(++p) != '\0')
 			options.servname = p;
 		    else if (*p != '\0') {
 			fprintf(stderr, "Malformed port \"%s\"\n", p);
@@ -269,7 +269,7 @@ static int parse_options(int *argcptr, char ***argvptr)
 			return -1;
 		    }
 
-		    if (*p == ':' && ++p != '\0')
+		    if (*p == ':' && *(++p) != '\0')
 			options.servname = p;
 		    else if (*p != '\0') {
 			fprintf(stderr, "Malformed port \"%s\"\n", p);
